src/fetcher.js

```diff
@@ -801,14 +806,14 @@ class Fetcher {
     }
 
     this.fetch(uri, options)
-      .then(result => {
+      .then(response => {
         if (userCallback) {
-          userCallback(true, null, result)
+          userCallback(response.ok, response.statusText || response.error, response.status)
```

Comment:
Just checking that response.status will play nicely with the update manager : 

https://github.com/linkeddata/rdflib.js/blob/master/src/update-manager.js#L995

Reply:
Good point!  ... maybe the callback is supposed to be (ok, message, response)
rather  than (ok, message, status)

Reply:
Yes, I think it used to be (ok, messagebody, xhr).  It should be (ok, message, response) Now .. new commit